doc/mgr/orchestrator.rst

```diff
@@ -144,6 +144,11 @@ Example command::
 
      ceph orch device zap my_hostname /dev/sdx
 
+.. note::
+    If you have used "--all-available-devices" in OSD creation, when you ``zap`` a device, immediately and automatically a new OSD will be created in the device.
```

Comment:
```suggestion
    If you use the `all-available-devices` option when creating an OSD, "zapping" a device automatically creates
    a new OSD in the device.

Reply:
I have edited all the below to say `all-available-devices` rather than `--all-available-devices` as you're talking about the parameter, and not showing how it is displayed in the command - rather, you do that later. But this call is up to you, just ensure it's consistent.

Reply:
In this case i think the original redaction is more clear. In your suggestion "zapping" seems to be what creates the new OSD, and that is not true. 
I have modified the redaction trying to have your suggestion into account and make more clear "what" is creating the new OSD.